test/jdk/javax/print/attribute/PageRangesException.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2001, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) 2001, 2025, Oracle and/or its affiliates. All rights reserved.
```
The year should be 2025 now.